src/item_factory.cpp

```diff
@@ -1002,29 +1004,44 @@ void Item_factory::load_book( JsonObject &jo )
     load_basic_info( jo, new_item_template );
 }
 
+void Item_factory::load( islot_comestible &slot, JsonObject &jo )
+{
+    slot.comesttype = jo.get_string( "comestible_type" );
+
+    jo.read( "charges", slot.def_charges );
+    jo.read( "tool", slot.tool );
+    jo.read( "quench", slot.quench );
+    jo.read( "brew_time", slot.brewtime );
+    jo.read( "addiction_potential", slot.addict );
+    jo.read( "fun", slot.fun );
+    jo.read( "stim", slot.stim );
+    jo.read( "healthy", slot.healthy );
+
+    if( jo.read( "spoils_in", slot.spoils ) ) {
+        slot.spoils *= 600; // JSON specifies hours so convert to turns
```

Comment:
`HOURS(x)` macro is better than hardcoding a multiplier. If length of an hour ever changes (for example, due to turn being shortened to 1 second, like Kevin said he wants it to work), hardcoded multipliers may be missed.